src/helpers/actions.ts

```diff
@@ -106,7 +106,7 @@ export async function getSpace(id: string, includeDeleted = false, network = DEF
     };
   }
 
-  const query = `SELECT settings, domain, deleted, flagged, verified, turbo, hibernated FROM spaces WHERE id = ? AND deleted in (?) LIMIT 1`;
+  const query = `SELECT settings, domain, deleted, flagged, verified, turbo, turbo_expiration, hibernated FROM spaces WHERE id = ? AND deleted in (?) LIMIT 1`;
```

Comment:
tests fail with `[Error: ER_BAD_FIELD_ERROR: Unknown column 'turbo_expiration' in 'field list']` maybe need to update schema

Reply:
updated. Also added an index in the schema file, can you add it to the database ?